### PR TITLE
Identify call doc fixes

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> For ActiveCampaign, Amplitude.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/analytics/amplitude.mdx [hunk 1]
```diff
@@ -534,15 +534,12 @@ The `identify` call lets you associate a user with their actions and capture all
...
 ```javascript
-rudderanalytics.identify({
-  userId: "userid",
-  anonymousId: "d80b66d5-b33d-412d-866f-r4fft5841af",
-  traits: {
+rudderanalytics.identify(
+    "userId", {
     email: "name@surname.com",
     name: "John Doe",
-    profession: "Student",
```
Comment (akashrpo) [inline destinations/analytics/amplitude.mdx:543]: We should keep existing data and just change it to the correct format.

File: destinations/marketing/activecampaign.mdx [hunk 2]
```diff
@@ -212,7 +203,7 @@ rudderanalytics.identify({
       Country: "USA",
       Hobbies: ["Cricket", "Tennis"],
     },
-    lists: [
+  lists: [
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply destinations/marketing/activecampaign.mdx:206]: done.

File: destinations/analytics/amplitude.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply destinations/analytics/amplitude.mdx:543]: sure. this is done.

**Review by akashrpo — APPROVED** [review]
LGTM

**Pull request merged**